### ISSUE TITLE
Unknown field type: 'V'

### ISSUE THREAD

**Issue by RomanKharin** [olemb/dbfread#6]
Some files created with FoxPro 9 can not be opened with dbfread.DBF("file.dbf")
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "~/dbfread/dbf.py", line 120, in __init__
    self._check_headers()
  File "~/dbfread/dbf.py", line 259, in _check_headers
    raise ValueError('Unknown field type: {!r}'.format(field.type))
ValueError: Unknown field type: 'V'
```

Seems to be DBF format slightly updated
https://msdn.microsoft.com/en-us/library/st4a0s68%28VS.80%29.aspx

**Comment by olemb** [comment]
dbfread should definitely support this. Do you have an example file I can
look at?

**Comment by bobintetley** [comment]
if I copy parseC in FieldParser.py and rename the copy parseV the error goes away and it doesn't seem to break anything.

**Comment by olemb** [comment]
Ah, that's great news. Thanks for letting me know.

I'll add support for this in the next release (which should be out any day
now).

**Comment by olemb** [comment]
I've added support for V fields to my develop branch.

@bobintetley I just added this to the parser class:

    parseV = parseC

Should work for now.

**Comment by olemb** [comment]
Not sure if you can comment on closed issues. Reopening just in case.

**Comment by olemb** [comment]
I just released 2.0.6 with support for V fields.
